src/smartCar/Webserver.h

```diff
@@ -1,9 +1,43 @@
 #ifndef WEBSERVER_H
 #define WEBSERVER_H
 
+
+#include <iostream>
+#include <HTTP_Method.h>
+#include <WebServer.h>
 #include <WiFi.h>
+#include <ESPmDNS.h>
+
+class Webserver {
```

Comment:
Inside the class you should have only things that you want to publicly expose or things you want to privately share among your functions.